package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "bedrock-express-browser-fixes",
+  "version": "0.0.1-0",
+  "description": "A Bedrock module for browser fixes.",
+  "main": "./lib",
+  "scripts": {
+    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require test/test-mocha.js test/10-api.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
+    "coverage-report": "nyc report",
+    "lint": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/bedrock-express-browser-fixes"
+  },
+  "keywords": [
+    "bedrock"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-express-browser-fixes/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-express-browser-fixes",
+  "dependencies": {
+    "esm": "^3.2.25",
+    "on-headers": "^1.0.2"
+  },
+  "peerDependencies": {
+    "bedrock": "^4.1.1",
+    "bedrock-express": "^3.2.0"
+  },
+  "directories": {
+    "lib": "./lib"
+  },
+  "devDependencies": {
+    "bedrock-express-browser-fixes": "file:.",
+    "bedrock-mongodb": "^8.2.0",
+    "bedrock-server": "^2.7.0",
+    "bedrock-test": "^5.3.2",
```

Comment:
This along witth `nyc` and possibly cross-env below look like they are borrowing from other test suite patterns that are not applicable here.  The test suite for this sort of module would be like this one: https://github.com/digitalbazaar/bedrock-account-http/tree/main/test

So, the immediate action is to remove all these bedrock devDependencies, and then later the test suite can be setup.